org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/AST.java

```diff
@@ -509,6 +525,11 @@ public final class AST {
 	 * deprecation warnings once JLS21 is deprecated
 	 */
 	static final int JLS22_INTERNAL = JLS22;
+	/**
+	 * Internal synonym for {@link #JLS23}. Use to alleviate
+	 * deprecation warnings once JLS22 is deprecated
```

Comment:
This should also be JLS23.

Reply:
Done. I fixed the JLS22_INTERNAL documentation too as it had the same issue.